easybuild/easyconfigs/p/ParaView/ParaView-5.11.1-foss-2022a-mpi-egl-osmesa.eb

```diff
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.11.1'
+versionsuffix = '-mpi-egl-osmesa'
```

Comment:
If we can make a fat build i think we can exclude the suffices completely. I think we should just drop the -mpi one as well (if someone wants to make a mpi-less version, let them add -nompi instead!)

We can combine this with bumping it to 2022b toolchain instead, which doesn't yet have a ParaView, and just treat it as the default build henceforth

Reply:
Ok, if we bump it to 2022b I will have to check all of the versions of the dependencies so that will take some more time, but I think there are GCC 12.2.0 easyconfigs of all the deps so that should be OK.